lib/api/apiUtils/object/coldStorage.js

```diff
@@ -34,6 +34,39 @@ function getAmzRestoreResHeader(objMD) {
     return undefined;
 }
 
+/**
+ * set archiveInfo headers to target header object
+ * @param {object} objMD - object metadata
+ * @returns {object} headers - target header object
+ */
+function setArchiveInfoHeaders(objMD) {
```

Comment:
I see no unit testing of this function, can we test it?

Reply:
it is tested in `objectHead` tests, which ensures we retrieve the right fields if/only when requested